### ISSUE TITLE
`watch` option is ignored by `@nx/vite:test`

### ISSUE THREAD

**Issue by lynxwint3r** [nrwl/nx#22392]
### Current Behavior

```pnpm nx test my-app --watch```

When I press `Enter` the command start, the tests are executed only once and then the script exit. Vitest doesn't enter in watch mode. 

The same behavior happens when I activat watch in `vite.config.ts`

```ts
export default defineConfig({
  // ...
  plugins: [vue(), nxViteTsPaths()],
  // ...

  test: {
    globals: true,
    cache: {
      dir: "../../node_modules/.vitest",
    },
    watch: true, // doesn't do anything
    environment: "jsdom",
    include: ["src/**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}"],
    reporters: ["default"],
    coverage: {
      reportsDirectory: "../../coverage/apps/my-app",
      provider: "v8",
    },
  },
```

It looks like the watch mode is totally ignored

### Expected Behavior

`vitest` should run in watch mode

### Steps to Reproduce

1. Execute `pnpx create-nx-workspace@latest`
   1. Enter `my-app` as workspace name
   2. Select `Vue` as stack
   3. Select `None` we only want `vue`
   4. Select `Integrated Monorepo`
   5. Application name: `my-app`
   6. `None` as e2e test runner
   7. Select `scss` for styling
   8. `Skip for now` for CI setup
   9. `Skip for now` for Remote caching
2. `cd my-app/`
3. `pnpm nx test my-app --watch`

### Nx Report

```shell
Node   : 20.11.1
OS     : darwin-arm64
pnpm   : 8.6.12

nx                 : 18.1.2
@nx/js             : 18.1.2
@nx/linter         : 18.1.2
@nx/eslint         : 18.1.2
@nx/workspace      : 18.1.2
@nx/devkit         : 18.1.2
@nx/eslint-plugin  : 18.1.2
@nrwl/tao          : 18.1.2
@nx/vite           : 18.1.2
@nx/vue            : 18.1.2
@nx/web            : 18.1.2
typescript         : 5.3.3
```

### Operating System

- [X] macOS
- [ ] Linux
- [ ] Windows
- [ ] Other (Please specify)

### Additional Information

I've seen the following issues that might be related to mine:
#20706 that should have been fixed by #20708.

But it looks like the issue is still there.

Did I missed or forget something in the setup or the config ? I don't know if I did something wrong but as you can see I just went with the default install.

If I did, let me know...

Thank you in advance

**Comment by thdk** [comment]
@lynxwint3r this is a duplicate of https://github.com/nrwl/nx/issues/22354

**Comment by FrozenPandaz** [comment]
Closing this as  a duplicate of https://github.com/nrwl/nx/issues/22354. Please follow for updates there.
